src/library/scala/ValueOf.scala

```diff
@@ -48,5 +51,5 @@ package scala
  * is required in the implementation of `+` it is provided at the call site via the
  * implicit argument `m` of type `ValueOf[M]`.
  */
-@scala.annotation.implicitNotFound(msg = "No singleton value available for ${T}.")
+@scala.annotation.implicitNotFound(msg = "No singleton value available for ${T}. (Eligible singleton types for `ValueOf` synthesis include literals and stable paths.)")
```

Comment:
```suggestion
@scala.annotation.implicitNotFound(msg = "No singleton value available for ${T}. Eligible singleton types for `ValueOf` synthesis include literals and stable paths.")
```

🤷‍♂️

Reply:
Semicolon!

Because they're inferred in code, we have extra left over for messaging.